code/modules/research/designs/bluespace_designs.dm

```diff
@@ -90,3 +90,13 @@
 	materials = list(MAT_METAL = 75000, MAT_GLASS = 37500, MAT_SILVER = 3000)
 	build_path = /obj/item/rpd/bluespace
 	category = list("Bluespace")
+
+/datum/design/bluespaceshotglass
+	name = "Bluespace Shot Glass"
+	desc = "For when you need to make the Bartender's life extra hell."
+	req_tech = list("bluespace" = 3, "toxins" = 6)
+	id = "bluespaceshotglass"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 7000, MAT_BLUESPACE = 2000)
```

Comment:
I feel like 3.5 sheets of metal and an entire bluespace crystal for a novelty with no real utility value is a bit steep, and might disincentivise this being used.

BS beakers are expensive because they hold 300U, this holds 50U and can be replaced with a normal small beaker or glass for chemistry needs.

---

Not saying you need to change the price, but it is food for thought.

Reply:
tbqh was not sure the exact conversion rate for points to sheets, and i knew it was (probably) gonna be outta wack

i mean feel free to throw out a reasonable price and i will commit it